components/brave_wallet/browser/internal/cardano_tx_decoder.h

```diff
@@ -27,8 +32,8 @@ class CardanoTxDecoder {
     SerializableTxInput& operator=(SerializableTxInput&&);
     ~SerializableTxInput();
 
-    std::array<uint8_t, 32u> tx_hash = {};
-    uint32_t index = 0;
+    std::array<uint8_t, kCardanoTxHashSize> tx_hash = {};
+    base::StrictNumeric<uint32_t> index = 0u;
```

Comment:
changes to use strict numeric seem independent of the other changes here, do we have a tracking issue for using this in general in wallet?